src/model/channel/message.rs

```diff
@@ -490,6 +490,18 @@ impl Message {
         http::send_message(self.channel_id.0, &map)
     }
 
+    /// Checks whether the message mentions the user of given Id.
+    pub fn mentions_user_id(&self, id: u64) -> bool {
```

Comment:
The reason I picked `u64` was to allow very direct usage if the Id is already known (compile-time).
E.g. `message.mentions_user_id(123)`
For every other "higher level" case of usage, I thought `mentions_user` would be sufficient, since it takes the effort of accessing the `UserId`.

Nonetheless, it should be even more explicit and yet still simple to construct `UserId` if wanted for direct Id usage, so I agree and changed accordingly.